junyub/code/burgerking.py

```
@@ -0,0 +1,77 @@
+    if current_count == prev_count:
+        break
+
+    prev_count = current_count
+
+    try:
+        more_button = wait.until(
+            EC.element_to_be_clickable((By.CSS_SELECTOR, "button.btn02.btn_more02"))
+        )
+        more_button.click()
+        time.sleep(1.5)
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)